src/views/Info.vue

```diff
@@ -283,7 +285,7 @@
 
          generate_predictions() {
              window.prediction_spreadsheet(
-                 new window.ExcelJS.Workbook(),
+                 new ExcelJS.Workbook(),
```

Comment:
a) this change isn't justified by the commit message, or the associated PR.  If it's important, it should go in a separate PR where we can discuss it
b) this `window`, and the one below, are there for ease of copy/pasting with code compiled with closure.  There are other solutions, but fair warning - it's a topic that will take a lot of time without generating much value

Reply:
a) I thought the change is self evident - since the library isn't loaded into the global scope but as a module instead, it can no longer be referenced using `window.ExcelJS`.
b) I'm unsure what you mean by ease of copy/pasting with code compiled with closure.

Reply:
Ah yes, my mistake.  The [Closure Compiler](https://developers.google.com/closure/compiler) requires some tricks to make it work, and this code is similar to some code in a repo which uses that compiler, but that's a problem for another day.